climada/util/test/test_config.py

```diff
@@ -61,6 +63,21 @@ def test_substitute(self):
         self.assertEqual(conf.a._root, conf._root)
         self.assertEqual(conf.a.str(), 'https://host/page.domain')
 
+    def test_set_logger(self):
+        #Check loggers are set to level
+        with self.assertLogs('climada', level='INFO') as cm:
+             with log_level('WARNING'):
+                logging.getLogger('climada').info('info')
+                logging.getLogger('climada').error('error')
+                self.assertEqual(cm.output, ['ERROR:climada:error'])
+        #Check other loggers are untouched
+        with self.assertLogs('matplotlib', level='DEBUG') as cm:
+            with log_level('ERROR', False):
+                logging.getLogger('climada').info('info')
+            logging.getLogger('matplotlib').debug('debug')
+            self.assertEqual(cm.output, ['DEBUG:matplotlib:debug'])
+
```

Comment:
Please adapt the tests accordingly if you would like to go with my `name_prefix` suggestion.

Reply:
Where would you put the test if the module is in the `__init__.py`? Also, in which `__init__.py` file would you put it? The one from the `util` module?